Gemfile

```diff
@@ -4,6 +4,7 @@ source "https://rubygems.org"
 gemspec
 
 gem "mixlib-shellout", "< 3.2.3"
+gem "chef-powershell", ">= 1.0.4"
 
 if Gem.ruby_version.to_s.start_with?("2.5")
```

Comment:
There's ruby 2.5 logic here, but you removed the ruby 2.5 testing. If we support it we should test it. If not remove it all and update the dep in the gemspec

Reply:
The gemspec had a requirement in there already. Removed this.